netlogo-gui/src/main/app/CodeTabsPanel.scala

```diff
@@ -131,4 +131,11 @@ class CodeTabsPanel(workspace:            GUIWorkspace,
       new AppEvents.SwitchedTabsEvent(previousTab, currentTab).raise(this)
     }
   }
+  java.awt.EventQueue.invokeLater(new Runnable() {
+
+    override def run(): Unit = {
+        codeTabContainer.toFront()
+        codeTabContainer.repaint()
+    }
+})
```

Comment:
The spacing here looks to be at 4 spaces, not 2.  The closing `})` is mis-aligned and should be indented 2 spaces to match up with the `(new Runnable() {` opening level.